apm-sniffer/apm-sdk-plugin/pom.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
I think you shouldn't change the line code.  because the line code has nothing to do with the future you commited.

Reply:
feature, not future, right? @ascrutae